doc/engineering-reference/src/simulation-models-encyclopedic-reference-002/air-system-compound-component-groups.tex

```diff
@@ -2085,7 +2085,7 @@ \subsubsection{Control Logic}\label{control-logic}
 
 \({\dot{V}_{exhaust,max}}\) is the exhaust fan maximum volumetric flow rate, user input (m\(^{3}\)/s)
 
-\({\rho_{Std}}\) is the density of air at standard temperature and pressure {[}dry air at 20°C{]} (m\(^{3}\)/kg)
+\({\rho_{Std}}\) is the density of air at standard temperature and pressure {[}dry air at 20$^\circ$C{]} (m\(^{3}\)/kg)
```

Comment:
I feel like we might have a degree macro defined somewhere, but this is fine.  Better than the embedded character.

Reply:
Sure. I remembered now that at some point some `\SI` type macro was used to address this. However, it seems that `^\circC` was used in a lot LaTeX documents (other than EnergyPlus LaTeX documentation) for degree signs. Anyway, I would think all these plain text replacements should be at least better than having the original Unicode characters.